contracts/interfaces/IZKPVerifier.sol

```diff
@@ -11,6 +11,13 @@ interface IZKPVerifier {
         bytes data;
     }
 
+    struct ProofStatus {
```

Comment:
is it ok to keep validator version and validator address in different structures? 
I understand that we can't modify ZKPRequest struct because of backward compatability, does it make sense to add validator address to ProofStatus ?